src/utils/utils.js

```diff
@@ -339,6 +339,20 @@ const linkSocial = async (token, provider, uid, email, done) => {
     return done(null, primary_account);
 };
 
+const sendRequestToken = (reqToken, res) => {
+    const payload = {
+        request_token: reqToken,
```

Comment:
The repo currently follows camel case throughout. It would be nice if camel case is followed everywhere. Kindly change the key value to `requestToken`.